Tests/MapboxCoreNavigationTests/PassiveLocationDataSourceTests.swift

```diff
@@ -97,6 +94,33 @@ class PassiveLocationDataSourceTests: XCTestCase {
         }
         wait(for: [locationUpdateExpectation], timeout: 5)
     }
+    
+    func testNoHistoryRecording() {
+        PassiveLocationDataSource.historyDirectoryURL = nil
+        
+        Navigator.shared.restartNavigator()
+        
+        let noHistoryExpectation = XCTestExpectation(description: "History should not be written on 'nil' path")
+        noHistoryExpectation.isInverted = true
+        PassiveLocationDataSource.writeHistory { _ in
+            noHistoryExpectation.fulfill()
```

Comment:
Not too critical, but I guess to make such test even more reliable we can check whether path to file is available in callback, but overall looks good to me.

Reply:
Current test is expected not to call the callback at all. But I've added additional check for writing history one.